students/junyoung/westagram/users/models.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+class User(models.Model):
+    name         = models.CharField(max_length=50)
+    email        = models.EmailField(max_length=100, unique=True)
+    password     = models.CharField(max_lenght=100)
```

Comment:
이후에 암호화된 password를 저장하기 위해 max_length를 넉넉하게 잡아주세요!
```suggestion
    password     = models.CharField(max_lenght=200)
```